src/pipecat/services/soniox/config.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from pipecat.transcriptions.language import Language
+
+
+class SonioxInputParams(BaseModel):
+    """Real-time transcription settings.
+
+    Attributes:
```

Comment:
For correct docstring auto-generation:

```suggestion
    Parameters:
```

Reply:
Can you update the Parameters to match what's in the class?